pyfar/dsp/dsp.py

```diff
@@ -1328,14 +1328,16 @@ def find_impulse_response_delay(impulse_response, N=1):
                     search_region_range[mask]-argmax, search_region[mask], N)
                 roots = np.roots(search_region_poly)
 
-        # Use only real-valued roots
-        if np.all(np.isreal(roots)):
-            root = roots[np.abs(roots) == np.min(np.abs(roots))]
-            start_sample = np.squeeze(lags[argmax] + root)
-        else:
-            start_sample = np.nan
-            warnings.warn(
-                f"Starting sample not found for channel {ch}", stacklevel=2)
+                # Use only real-valued roots
+                if np.all(np.isreal(roots)):
+                    root = roots[np.abs(roots) == np.min(np.abs(roots))]
+                    start_sample[idx] = np.squeeze(lags[argmax] + root)
+                else:
+                    start_sample[idx] = np.nan
+                    warnings.warn('Starting sample not found for channel '
+                                  f'{ch}', stacklevel=2)
+            else:
+                start_sample[idx] = np.nan
```

Comment:
Hm, weird - did you touch the indention here and below? The browser interface shows it as changes, the VSCode interface does not...

Reply:
This was in one commit, which was reverted somehow (now idea but looks fine now)